release-team/role-handbooks/emeritus-adviser/README.md

```diff
@@ -4,22 +4,33 @@
 
-As a returning RT member, the EA is also there to supply the Release Lead with historical information about how things were handled in prior releases, and serve as a sounding board when both the Release Lead and Lead Shadows are facing a dilemma.
+Time commitments for the EA are relatively light, with the largest time commitment in the two-week period where Shadows are being selected.  However, the EA role starts 2-3 weeks before the release cycle officially begins, and continues for 1-2 weeks after it ends.
 
-## Requirements
+EAs are not chosen as part of the regular RT selection process.  Instead, interested candidates should contact the outgoing EA and the incoming Release Lead 2 weeks before the final prior release.  Most of the time, the outgoing EA will solicit suitable candidates.
 
-The EA is a returning senior Release Team member, ideally a former Release Team Lead. If not a former Release Team Lead, the EA should have been on the Release Team for no less than two release cycles, serving as a Role Lead for at least one of those cycles.  An ideal Emeritus Adviser is inherently suited to people management, has shown sustained interest in the health and growth of SIG Release, and has a commitment to Release Team diversity and helping others ascend the [Contributor Ladder](https://git.k8s.io/community/community-membership.md).
+The EA for the next release team should be selected ahead of opening up nominations for the next release cycle, and the incoming EA will be assigned that issue. 
 
-Time commitments for the EA are relatively light, with the largest time commitment in the two-week period where Shadows are being selected.  However, the EA role starts 2-3 weeks before the release cycle officially begins, and continues for 1-2 weeks after it ends.
+Due to the nature of the role, the EA has no shadows.  The job may, however, be split between more than one person.
 
-Due to timing (see below) EAs are not chosen as part of the regular RT selection process.  Instead, interested candidates should contact the outgoing EA and the incoming Release Lead 2 weeks before the final prior release.  Most of the time, the outgoing EA will solicit suitable candidates.
+## Responsibilities
+Responsibilities for the EA are to help enable the Release Team. Activities the EA is responsible for are 
 
-Due to the nature of the role, the EA has no shadows.  The job may, however, be split between more than one person.
+* Tracking the former release retrospective AIs 
+* Creating the incoming release shadow surveys 
+* Assisting release role leads in shadow selection 
+* Running a general shadow orientation for folks, explaining the basics of the release team 
+* Checking in on leads and shadows throughout the release and offering support to them 
+* Conducting Shadow experience surveys throughout the release cycle and a lead/mentor questionaire at the end of the cycle
+* Helping facilitate the release retro (if no other facilitator can be found) 
```

Comment:
Change the verb form at the beginning to Track, Create, Assist etc to be consistent with the rest of the handbook

Reply:
updated verb-age